javaslang/src/main/java/javaslang/collection/List.java

```diff
@@ -1004,7 +1004,7 @@ default T peek() {
     @Override
     default List<T> removeAll(Iterable<? extends T> elements) {
         Objects.requireNonNull(elements, "elements is null");
-        final List<T> removed = List.<T> ofAll(elements).distinct();
+        final List<T> removed = List.ofAll(elements).distinct();
```

Comment:
We added some of these type hints because of Eclipse errors. Most of us use IntelliJ IDE.
Let's remove it though.